src/query/storages/common/table_meta/src/meta/v5/snapshot.rs

```diff
@@ -28,18 +27,24 @@ use crate::meta::format::encode;
 use crate::meta::format::read_and_deserialize;
 use crate::meta::format::MetaCompression;
 use crate::meta::monotonically_increased_timestamp;
+use crate::meta::snapshot_id::new_snapshot_id_with_timestamp;
 use crate::meta::trim_timestamp_to_micro_second;
 use crate::meta::v2;
 use crate::meta::v3;
 use crate::meta::v4;
-use crate::meta::v5::Statistics;
 use crate::meta::ClusterKey;
 use crate::meta::FormatVersion;
 use crate::meta::Location;
 use crate::meta::MetaEncoding;
 use crate::meta::SnapshotId;
+use crate::meta::Statistics;
 use crate::meta::Versioned;
 
+pub type TableVersion = u64;
+// information to generate snapshot file name
+pub type SnapshotLocationInfo = Option<(SnapshotId, FormatVersion, Option<TableVersion>)>;
+
+/// The structure of the segment is the same as that of v2, but the serialization and deserialization methods are different
 #[derive(Serialize, Deserialize, Clone, Debug)]
```

Comment:
Maybe we can use the new Statistics https://github.com/datafuselabs/databend/blob/v1.2.184-nightly/src/query/storages/common/table_meta/src/meta/v5/statistics.rs here